common/mod/ngv/cadifus.mod

```diff
@@ -363,7 +364,18 @@ BREAKPOINT {
     :i=0
     caerAll=caer[0]
 
-    if (glu2>0){
+    is_glu2_assigned = 0
+: here we want to check glu2 assignment. checking _p_glu2 ends in 
+: RuntimeError: hocobj_call error: generic_data_handle{cont=GlutReceive 
+: glut row=0/129 type=double*}::literal_value<void*> cannot be called on 
+: a handle [that was] in modern mode
+VERBATIM
+    if (_ppvar[0].get<double*>()) {

```

Comment:
is `_ppvar[0]` going to be stable? or does it also depend on how MOD files are translated and compiled?

Reply:
sincerely I do not know. Probably it is stable if we do not change other code in cadifus. There is an issue for another case of this exact problem in neuron and is still open. This is my way of avoiding it. I do not see atm another way except:
- forcing all the pointers to be valid
- changing neuron

Reply:
fair, I don't know this stuff at all either.
If we have no choice, can you link the other issues as comments, so if someone checks them and they are solved, then we can update to whatever the fix is?  That way there is a foothold to finding out more info.

Reply:
I expanded the comment. Let me know what you think